/*
    try {
         Try Block to check for errors.
    }
    catch(err) {
          Catch Block to display errors.
    }
*/
console.log("Hello 1");
console.log("Hello 2");
// console.log(a); Error
try {
  console.log(a);
}
catch(err) {
  console.log("Somthing Wrong");
}
console.log("Welcome 1");
console.log("Welcome 2");
